### ISSUE TITLE
The "Show password" cannot be accessed

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-tv#2590]
## Steps to reproduce
1. Launch FF.
2. Add a valid email address.
3. Tap on the submit button.
4. Enter the password.
5. Try to select and tap on the "Show password" button.
### Expected behavior
The password is revealed.
### Actual behavior
Nothing is displayed and the button doesn't function.
### Device information
* Fire TV device: **`Fire TV Stick 4k and 2nd Gen Stick`**
* Latest affected Firefox version: **`Firefox TV v4.1 LAT2`**
* Earliest affected Firefox version: **`Firefox TV v4.1 LAT2`**
* Turbo Mode: **`On And Off`**

**Comment by mcomella** [comment]
@Baron-Severin Should we forward this to the FxA team?

**Comment by athomasmoz** [comment]
Yes, let's send to FxA team. Would like to get their opinion on priority based on how often the feature is used. I don't think it needs to block our account log in release

**Comment by athomasmoz** [comment]
FxA issue logged https://github.com/mozilla/fxa/issues/2022
